### PR TITLE
Adds the ability to call in a tactical hood ian via uplink

### PR DESCRIPTION
**Pull request by dragoslava**
- Adds a new Ian sprite
- Adds a new buyable mob for antags to use, Ian's cousin that dress's like a gangster and likes to cause as much problems as its owner. Cost's 4 TC

### CODE REVIEW

**Comment by BotBOREALIS** [comment]
Icons with diff:
<details><summary>icons/mob/dog.dmi</summary>

Key | Old | New | Status
--- | --- | --- | ---
iangang||iangang|Created
iangang[1]||![iangang[1]](http://images.aurorastation.org/images/6cbfc544647d6888498e3e38f30fcc1b.png)|Created
iangang[2]||![iangang[2]](http://images.aurorastation.org/images/4b6e47225b6cd688b5e726f4a7857983.png)|Created
iangang[3]||![iangang[3]](http://images.aurorastation.org/images/5b6e80bcea0e33db59f5608f12f345f0.png)|Created
iangang_dead||iangang_dead|Created
</details>

**Comment by BurgerLUA** [comment]
What does the dog even do exactly?

**Comment by dragoslava** [comment]
Acts as a pet/attack dog. Its pretty simple really

**Comment by BurgerLUA** [comment]
A 4TC meme guard dog.
Yeah no thanks.

File: code/datums/uplink/devices and tools.dm [hunk 1]
```diff
@@ -9,6 +9,12 @@
 	item_cost = 2
 	path = /obj/item/weapon/storage/toolbox/syndicate
 
+
+/datum/uplink_item/item/tools/toolbox
```
Comment (Sindorman) [inline code/datums/uplink/devices and tools.dm]: Why is this a toolbox?

**Comment by dragoslava** [comment]
Closing this till I can get help submiting it properly

**Pull request closed without merging**